ci-operator/config/openshift/cluster-samples-operator/openshift-cluster-samples-operator-release-4.11.yaml

```diff
@@ -51,6 +51,74 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: check-aws-no-capabilities
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
+  steps:
+    cluster_profile: aws
+    env:
+      BASELINE_CAPABILITY_SET: None
+    test:
+    - as: test
+      cli: latest
+      commands: |
+        IMAGESTREAM_ALLOWED_LIST="cli cli-artifacts driver-toolkit installer installer-artifacts must-gather oauth-proxy tests tools"
+        TEMPLATE_ALLOWED_LIST=""
+        function exists_in_list() {
+          LIST=$1
+          DELIMITER=$2
+          VALUE=$3
+          LIST_WHITESPACES=`echo $LIST | tr "$DELIMITER" " "`
+          for x in $LIST_WHITESPACES; do
+            if [ "$x" = "$VALUE" ]; then
+              return 0
+            fi
+          done
+          return 1
+        }
+
+        function check_present_list_is_correct() {
+          CURRENT_LIST=("$@")
+          ((last_idx=${#CURRENT_LIST[@]} - 1))
+          KIND=${CURRENT_LIST[last_idx]}
+          unset CURRENT_LIST[last_idx]
+          for i in "${CURRENT_LIST[@]}";
+            do
+              if [ "$KIND" == "imagestreams" ] ; then
+                if exists_in_list "$IMAGESTREAM_ALLOWED_LIST" " " $i; then
+                  echo "imagestream $i is present, which is expected"
+                else
+                  echo "$i imagestream should not be present"
+                  return 1
+                fi
+              elif [ "$KIND" == "templates" ] ; then
+                if exists_in_list "$TEMPLATE_ALLOWED_LIST" " " $i; then
+                  echo "template $i is present, which is expected"
+                else
+                  echo "$i template should not be present"
+                  return 1
+                fi
+              else
+                echo "Unknown kind passed as an argument"
+                return 1
+              fi
+            done
+        }
+
+        function assert_empty() {
+          KIND="$1"
+          NAMES="$(oc -n openshift get -o jsonpath='{.items[*].metadata.name}' "${KIND}")"
```

Comment:
If you want `NAMES` to be a Bash array, so `"${NAMES[@]}"` will expand to multiple arguments, you should use:

```sh
NAMES=($(oc -n openshift get -o jsonpath='{.items[*].metadata.name}' "${KIND}"))
```

Reply:
changed name variable to be a Bash array and did the subsequent changes as well. @wking could you please review once again?